disasterinfosite/models.py

```diff
@@ -263,6 +269,62 @@ def default_display_name(sender, instance, *args, **kwargs):
         instance.display_name = instance.name
 
 
+# looks up a point in a raster
+# algorithm for this function taken from the django-raster project version 0.6 at
+# https://github.com/geodesign/django-raster/blob/master/raster/utils.py
+# and is covered by the following licence:
+'''
+Copyright (c) Daniel Wiesmann and Mike Flaxman
+All rights reserved.
+
+Redistribution and use in source and binary forms, with or without modification,
+are permitted provided that the following conditions are met:
+
+    1. Redistributions of source code must retain the above copyright notice,
+       this list of conditions and the following disclaimer.
+
+    2. Redistributions in binary form must reproduce the above copyright
+       notice, this list of conditions and the following disclaimer in the
+       documentation and/or other materials provided with the distribution.
+
+    3. Neither the name of the author nor the names of other contributors may
+       be used to endorse or promote products derived from this software
+       without specific prior written permission.
+
+THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND
+ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED
+WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT OWNER OR CONTRIBUTORS BE LIABLE FOR
+ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES
+(INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES;
+LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON
+ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+(INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS
+SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+'''
+def rasterPointLookup(rst, lng, lat):
+    pnt = OGRGeometry('POINT({0} {1})'.format(lng, lat), srs=rst.srs)
+
+    # if the point is outside the raster's extent, we can save time and just return None here
+    bbox = OGRGeometry.from_bbox(rst.extent)
+    bbox.srs = rst.srs
+    if not pnt.intersects(bbox):
+        return -1
```

Comment:
Note to self: whatever we use here as NODATA should be mentioned in the readme so that people know to avoid it as a data value.  And it's worth me spending a few minutes looking at examples to at least settle on something that's commonly used as such anyway.

Reply:
....aaaaactually... I think I can have it use the NODATA value that's set for `rst`.  That should be safer.

Reply:
that'd be the best if you can!